packages/mock-wrapper/package.json

```
@@ -22,7 +22,10 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@emotion/core": "^10.0.16",
```

For safety's sake I've been using the same @emotion/core version across packages and using the 2019-template package as my reference. That one is at ^10.0.10. It shouldn't matter because they both have the `^`. @jaronheard, any preference here?

---

I generally do the same thing as @Dianna. It shouldn't matter but might as well keep it consistent.
```suggestion
    "@emotion/core": "^10.0.10",
```